README.md

```diff
@@ -22,7 +22,7 @@ For using this plugin on iOS, there are some additional requirements:
 
 # Installing
 
-    $ cordova plugin add https://github.com/fxe-gear/cordova-plugin-ble-central.git
+    $ cordova plugin add https://github.com/flemmingdjensen/cordova-plugin-ble-central.git
```

Comment:
It would be better to keep the plugin  in the same repository, since there may be projects which depends on this URL.

Reply:
If you find this objective valid/useful, we will solve it in next PR.